### ISSUE TITLE
Left click stops working after pressing windows key

### ISSUE THREAD

**Issue by BurningFlames2314** [ppy/osu#17945]
### Type

Game behaviour

### Bug description

when holding left click using tablet and pressing the windows key to minimise it, when reopening os lazer back, the left click seems to be stuck down. This bug doesnt seem to appear when doing it with the mouse tho. Using keyboard still works.

### Screenshots or videos

https://user-images.githubusercontent.com/104247330/164883180-16b4ed6b-4473-4167-a178-2e58647e8cd4.mp4

### Version

2022.418.0

### Logs

[legacy-ipc.log](https://github.com/ppy/osu/files/8546390/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/8546391/network.log)
[performance.log](https://github.com/ppy/osu/files/8546392/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8546393/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8546394/updater.log)
[database.log](https://github.com/ppy/osu/files/8546395/database.log)

**Comment by peppy** [comment]
Can you confirm that this only happens with your tablet?

Also, can you try toggling the in-game tablet support on and off and see if it happens in either or both cases?

**Comment by BurningFlames2314** [comment]
yup I just tried it out a bunch of times again. Doesn't matter if tablet support is on or off, the m1 button gets stuck if I tab out of lazer holding m1 with the pen and switching to mouse before reopening lazer again.

Also tried out setting m1 and m2 to my pen buttons. m2 doesn't seem to cause the bug but m1 still does.

Doing mouse then switching to tablet does not create the bug.

Interesting note: When I tab out while holding m1 using tablet... and tab back in, no matter what, m1 is "stuck" being held down.  But as long as the mouse wasn't touch, it can be "unstucked" by just pressing m1 again using the pen. (pressing m1 with the pen while lazer is tabbed out does nothing, have to press while currently in game to get it unstuck).

If the m1 is currently "stuck" and I don't get it unstuck, any mouse movements or actions causes the cursor to get locked holding down m1... (both when lazer is in background and when active)

Not very knowledgeable in these kind of things... but from what I can tell after examining what is going on is that the m1 button gets frozen and the mouse causes it to freak out? idk what I'm talking about, I'm talking nonsense right now.

I just tested this on another computer as well and the bug seems to appear as well. Though it might just be my tablet being funky so I'm not sure... I'm using a Wacom Intuos BT M
